src/Components/StepTwo/StepTwo.js

```diff
@@ -1,63 +1,284 @@
-                    <input type="text" placeholder="Address"/>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"Address"}
+                        name={"addressLine1"} 
+                        value={inputs.address.addressLine1} 
+                        onChange={handleChange}/>
+                    </div>
+                    <div className="addressLine">
+                    <p>Address Line 2</p>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"Address"}
+                        name={"addressLine2"} 
+                        value={inputs.address.addressLine2} 
+                        onChange={handleChange}/>
+                    </div>
                 </div>
-                <div className="addressLine">
-                    <p><span>*</span> Address Line 2</p>
-                    <input type="text" placeholder="Address"/>
-                </div>
-            </div>
 
-            <div className="homeCityInput">
-                <div className="city">
-                    <p><span>*</span> City/Town</p>
-                    <input type="text" name="" id="" placeholder="City"/>
-                </div>
-                <div className="state">
+                <div className="homeCityInput">
+                    <div className="state">
                     <p><span>*</span> State</p>
-                    <input type="text" name="" id="" placeholder="State"/>
-                </div>
-                <div className="zip">
+                    <Input 
+                        type={"text"} 
+                        placeholder={"State"}
+                        name={"state"} 
+                        list={"state"} 
+                        value={inputs.address.state} 
+                        onChange={handleChangeAddressState}/>
+                    <datalist id={"state"}>
+                    {dataFetch.map((state, index) => (
+                    <option value={state.name || state.code} 
+                    key={`${state.name}`+ index} />
+                      ))}
+                    </datalist>
+                    </div>
+                    <div className="city">
+                    <p><span>*</span> City/Town</p>
+                    <Input 
+                        type={"text"} 
+                        placeholder={"City"}
+                        name={"city"} 
+                        list={"city"} 
+                        disabled={inputs.address.state === "" ? true : false} 
+                        value={inputs.address.city} 
+                        onChange={handleChange}/>
+                     <datalist id={"city"}>
+                      { tla && dataCities.map((city, index) => {
+                        return (
+                          <option 
+                          value={city.name || city.code} 
+                          key={`${city.name}`+ index}> </option>
```

Comment:
Is there a reason to combine template string with "+" operator?

Reply:
I just wanted to concatenate city name with index, because there are some cities with the same them, therefore key can not be unique. In case you asked me this just for a question mark, I deleted it and the result stayed the same :)